### PR TITLE
chore(macros): Remove 'Xref_cssinitial', 'Xref_cssinherited', 'Xref_csscomputed', 'NonStandardBadge'

### PR DESCRIPTION
**Pull request by bsmth**
### Description

This PR removes the only occurrences of the following macros in content:

- `{{Xref_cssinitial}}`
- `{{Xref_cssinherited}}`
- `{{Xref_csscomputed}}`
- `{{NonStandardBadge}}`

There are no other uses of these macros and they can be replaced with regular Markdown links.

In the case of `{{NonStandardBadge}}`, replaced with `{{non-standard_inline}}` which appears to have the same effect.

### Motivation

Removing unnecessary macros

### CODE REVIEW

File: files/en-us/web/css/-webkit-mask-box-image/index.md [hunk 1]
```diff
@@ -77,10 +77,10 @@ Border repeat styles, when included, are interpreted in the order of `<repeat-x>
 
 ## Formal definition
 
-- {{ Xref_cssinitial() }}: `none`
+- [Initial value](/en-US/docs/Web/CSS/initial_value): `none`
 - Applies to: all elements
-- {{ Xref_cssinherited() }}: no
-- {{ Xref_csscomputed() }}: as specified
+- [Inherited](/en-US/docs/Web/CSS/Inheritance): no
+- [Computed value](/en-US/docs/Web/CSS/computed_value): as specified
```
Comment (dipikabh) [inline files/en-us/web/css/-webkit-mask-box-image/index.md:83]: Maybe we can replace these lines with the `{{CSSInfo}}` macro?
Some of the other `-webkit-mask-*` properties are using `{{CSSInfo}}`, so might work here.

**Review by dipikabh — CHANGES_REQUESTED** [review]
Thanks, @bsmth! I've added a comment.

File: files/en-us/web/css/-webkit-mask-box-image/index.md (hunk 1 above)
Comment (bsmth) [reply files/en-us/web/css/-webkit-mask-box-image/index.md:83]: I get `Value not found in DB!`. And it looks like it's missing from mdn/data:

https://github.com/mdn/data/blob/af8a84ad4bdc9ea92fe74790eed43712c5668f6b/css/properties.json#L1362-L1378

Is it worth adding it, do you reckon?

Comment (dipikabh) [reply files/en-us/web/css/-webkit-mask-box-image/index.md:83]: Yeah, not worth it updating `mdn/data` for this non-standard property.

Plus, I found this odd: https://github.com/mdn/data/blob/af8a84ad4bdc9ea92fe74790eed43712c5668f6b/css/properties.json#L1341-L1348. For example, it does not include `-webkit-mask-composite`. No point chasing this one.

In any case, the note on [-webkit-mask-box-image](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-mask-box-image) page does tell readers to refer to the [mask-border](https://developer.mozilla.org/en-US/docs/Web/CSS/mask-border) property.
This is good as is I think.

**Review by dipikabh — APPROVED** [review]
Thanks, @bsmth 👍

**Comment by bsmth** [comment]
Thank you!

**Comment by adeibnugandiSH** [comment]
Hello Cs

**Pull request merged**